core/controllers/creator_dashboard.py

```diff
@@ -233,12 +233,20 @@ def _round_average_ratings(rating):
 
             subscribers_list.append(subscriber_summary)
 
+        user_settings = user_services.get_user_settings(
+            self.user_id, strict=False)
+        if user_settings.creator_dashboard_display:
+            display_preference = user_settings.creator_dashboard_display
+        else:
+            display_preference = 'card'
```

Comment:
This should be pulled out into a constant somewhere, e.g. CREATOR_DASHBOARD_DISPLAY_PREF_CARD.

Also, will the None case happen, if we have a default specified in the storage layer?